.eslintrc

```diff
@@ -72,11 +64,15 @@
         "no-use-before-define": "off",
         "object-curly-newline": "off",
         "instawork/error-object": "off",
-        "@typescript-eslint/no-explicit-any": "off",
-        "implicit-arrow-linebreak": "off",
```

Comment:
> Removed following ignored rules by updating the code
> max-len
> implicit-arrow-linebreak
> @typescript-eslint/no-explicit-any

Does this mean all arrow line breaks errors are resolved?

Reply:
Yes